src/main/java/io/fabric8/elasticsearch/plugin/KibanaIndexMode.java

```diff
@@ -21,12 +21,15 @@
      * The setting that determines the kibana index is used by users.  Valid values are one of the following:
      * 
      *   * unique (Default) - Each user gets a unique index for kibana visualizations (e.g. .kibana.USER_UUID)
-     *   * ops_shared       - Users who are in an ops role will share an index (e.g. kibana) while non ops users will 
+     *   * shared_ops       - Users who are in an ops role will share an index (e.g. kibana_ops) while non ops users will 
      *                        have a unique index (e.g. .kibana.USER_UUID)
+     *   * shared           - Users who are in an ops role will share an index (e.g. kibana_ops) while non ops users will 
+     *                        share the default index (e.g. .kibana)                      
      */
     static final String OPENSHIFT_KIBANA_INDEX_MODE = "openshift.kibana.index.mode";
     
     static final String UNIQUE = "unique";
     static final String SHARED_OPS = "shared_ops";
+    static final String SHARED = "shared";
```

Comment:
We may wish to consider changing this name to something like 'shared_non_ops' in the event we wish to make kibana index 'shared' across all users.  That was my thought originally for 'shared'.  It would be interesting how a full 'shared' mode might exist given I assume we still might restrict non ops users to specific indicies

Reply:
the "shared" is enabling dashboard sharing for ops and non-ops (users) do you think we need a shared for both?

Reply:
I don't know but should we ever (and i know thats not very agile to think of it know) but then shared for everyone would be.... 'global shared'?

Reply:
ok you are right, but "shared_non_ops" is not the right key since, it enable the sharing for both, may we find another key ;)

Reply:
I am open to another suggestion, any 'good' one escapes me at the moment